src/documentation/gtag.ts

```diff
@@ -28,37 +29,37 @@ export class GTAGDocumentationProvider implements GTA3DocumentationProvider {
                 examples: [],
             };
 
-            let $ = cheerio.load(body);
-            $(".opcodeinfo > tbody > tr > td").each((i, elem) => {
-                if(i == 0) { // Description
-                    let elemHtml = $(elem).html();
-                    if(elemHtml) {
-                        result.shortDescription = $(`<span>${elemHtml.split(/<br>/)[0]}</span>`).text().split('.')[0]
-                        result.longDescription = this.htmlToMarkdown(elemHtml);
-                    }
-                } else if(i == 1) { // Parameters
-                    let params = $(elem).html().split(/<br>/g)
-                    result.args = params.map(p => {
-                        let m = $(p).text().match(/^\d+\)\s*(?:\(Returned\))?\s*([^\(]*)/);
-                        return { type: null, description: (m && m[1]) || null };
-                    });
-                } else if(i == 2) { // Games 
-                    result.games = $(elem).children("img").map((_, img) => {
-                        let src = $(img).attr('src');
-                        if(src.endsWith("/sa.gif")) return GameDoc.SanAndreas;
-                        if(src.endsWith("/vc.gif")) return GameDoc.Miami;
-                        if(src.endsWith("/iii.gif")) return GameDoc.Liberty;
-                        return null;
-                    }).filter(x => x != null).toArray();
-                }
-            });
+            let info = json.parse(body);
+            if (!info.hasOwnProperty("status") || info.status != "200")
+                Promise.reject("GTAG OpcodeDB API request failed (status: "+info.status+")");
```

Comment:
Not 100% sure, but I think the promise object should be returned, i.e. `return Promise.reject(...);`?